smoke/IotEdgeQuickstart/details/TwinTestConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace IotEdgeQuickstart.Details
+{
+    using System.Collections.Generic;
+    using Microsoft.Azure.Devices.Shared;
+    using Newtonsoft.Json;
+
+    public class TwinTestConfiguration
+    {
+        [JsonProperty(PropertyName = "moduleId")]
+        public string ModuleId { get; set; }
+
+        [JsonProperty(PropertyName = "properties")]
+        public TwinProperties Properties;
```

Comment:
add {get; set;}

Reply:
Interesting how it worked without it. :) Done.